go/tools/builders/protoc.go

```diff
@@ -26,6 +28,16 @@ import (
 	"strings"
 )
 
+type FileInfo struct {
```

Comment:
I think it's a little confusing this has the same name as `os.FileInfo`. It doesn't need to be public though, right? Maybe `genFileInfo` would be a better name.

Also, consider making the fields non-public.

Reply:
Meh, this entire directory is "private", it's all built directly to binaries and not libraries, and it's all in package main, so it's not like it can be imported.